openedx/features/pakx/lms/pakx_admin_app/constants.py

```diff
@@ -4,3 +4,5 @@
 STAFF = 2
 TRAINING_MANAGER = 3
 LEARNER = 4
+
+DEFAULT_REGISTRATION_PASSWORD = "12345"
```

Comment:
shouldn't we create a random password for each user?

Reply:
+1